riptide-failsafe/src/main/java/org/zalando/riptide/failsafe/FailsafePlugin.java

```diff
@@ -29,11 +31,11 @@
     private final RetryListener listener;
 
     public FailsafePlugin(final ScheduledExecutorService scheduler) {
-        this(scheduler, NEVER, new CircuitBreaker(), RetryListener.DEFAULT);
+        this(scheduler, NEVER, null, RetryListener.DEFAULT);
```

Comment:
It is unfortunate that `CircuitBreaker` is not an interface and there is nothing like `CircuitBreaker.NOOP`